lib/config/index.js

```diff
@@ -181,6 +181,7 @@ switch (config.imageUploadType) {
     break
   default:
     config.allowedUploadMimeTypes = [
+      '*',
```

Comment:
We should be careful about this. I would still recommend to limit the uploads a bit to "passive" documents. Just putting a wildcard here, appears a bit too broad to me.

The alternative would be to blacklist mimetypes, but that makes things even more difficult.

I also think we need to add a case for lutim in here as I don't expect lutim to support generic files.